File: src/test/java/org/assertj/core/data/TolkienCharacter.java
package org.assertj.core.data;

public class TolkienCharacter {

  public static TolkienCharacter of(String name, Integer age, Race race) {

	return new TolkienCharacter(name, age, race);
  }

  public static enum Race {
	HOBBIT,
	MAIA,
	ELF,
	DWARF,
	MAN
  }

  public final String name;
  public final Integer age;
  public final Race race;

  private TolkienCharacter(String name, Integer age, Race race) {

	this.name = name;
	this.age = age;
	this.race = race;
  }

  public String getName() {

	return name;
  }

  public Integer getAge() {

	return age;
  }

  public Race getRace() {

	return race;
  }
}

File: assertj-tests/assertj-integration-tests/assertj-core-kotlin/src/test/kotlin/org/assertj/tests/core/kotlin/testkit/TolkienCharacter.kt
package org.assertj.tests.core.kotlin.testkit

data class TolkienCharacter(
  val name: String,
  val age: Int,
  val race: Race) {

  enum class Race {
    HOBBIT, MAIA, ELF, DWARF, MAN, DRAGON
  }
}
